bigtable/quickstart/main.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Google LLC
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START bigtable_quickstart]
+// Quickstart is a sample program demonstrating use of the Cloud Bigtable client
```

Comment:
add a single line above, otherwise this becomes part of the package comment.

Reply:
done.